examples/gno.land/r/matijamarjanovic/home/config.gno

```diff
@@ -0,0 +1,70 @@
+package home
+
+import (
+	"errors"
+	"std"
+)
+
+var (
+	mainAddr   std.Address // tony's main address
+	backupAddr std.Address // backup address
+
+	errorInvalidAddr string
+	errorUnauthorized string
+)
+
+func init() {
+	mainAddr = "g1ej0qca5ptsw9kfr64ey8jvfy9eacga6mpj2z0y"
+	errorInvalidAddr = "config: invalid address"
+	errorUnauthorized = "config: unauthorized"
+}
```

Comment:
This is one of the things I did the way Leon did, do not know the reason but I've seen it done this way so I did the same thinig.

Reply:
Currently, top level initializations & `init` are the same; it can be done either way. For errors, you should use `errors.New` as @Maelkum suggested. 

Also, would be good to explicitly cast the string of the address to `std.Address`

Reply:
Initialized errors and cast the address in these commits: 3dc2c4e3009ded5948fbaa7c4608fb0e096428d5 and 0c35c090d8f40176785f36f692bab3495255bd75